woudc_data_registry/product/uv_index/uv_index_generator.py

```diff
@@ -357,6 +396,7 @@ def compute_uv_index(ipath, extcsv, dataset, station,
             if instrument_name.lower() == 'brewer':
                 try:
                     intcie = extcsv.extcsv[global_summary_t]['IntCIE'][0]
+                    print(f'IntCIE: {intcie}')
```

Comment:
```suggestion
                    LOGGER.debug(f'IntCIE: {intcie}')
```
I'm guessing this should be a `debug`?